TYPO3.Flow/Classes/TYPO3/Flow/Http/Request.php

```diff
@@ -373,6 +417,136 @@ public function getContent($asResource = false)
     }
 
     /**
+     * Set a trusted proxy header.
+     *
+     * This is primarily useful for testing. If you want to set trusted headers, use the configuration setting
+     * "TYPO3.Flow.http.trustedProxies.headers" instead.
+     *
+     * @param string $type One of the HEADER_* constants
+     * @param string $name The header name to trust or an empty string to not trust this header
+     * @return void
+     */
+    public static function setTrustedProxyHeader($type, $name)
```

Comment:
Ok, got you
> Not sure if there's really a plausible use case outside of testing, but it could be.
I call YAGNI ;)  lets inject this from the tests and implement it once actually needed.

But if we decide to keep it, we should probably throw an exception for a non-allowed $type instead of silently ignoring that case